### PR TITLE
Add technical content as per the codebase repository

### PR DESCRIPTION
**Pull request by ParthBrahmbhatt**
## Purpose of this PR

This PR add the checkbox option related technical documentation as per the magento develop version codebase repository.

## Affected URLs

- v2.3 https://devdocs.magento.com/guides/v2.3/graphql/reference/customizable-option-interface.html

## Links to source code

- v2.3 https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/CatalogGraphQl/etc/schema.graphqls#L423

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by keharper** [comment]
@ParthBrahmbhatt The `CustomizableMultipleOption` and `CustomizableMultipleValue` types were also added for 2.3.2. Please add these to your PR.

File: guides/v2.3/graphql/reference/customizable-option-interface.md [hunk 1]
```diff
@@ -140,6 +141,46 @@ Attribute | Type | Description
...
+`sort_order` | Int | The order in which the option is displayed
+`title` | String | The display name for this option## CustomizableRadioOption object
+
+`CustomizableRadioOption` contains information about a set of radio buttons that are defined as part of a customizable option.
+
+Attribute | Type | Description
+--- | --- | ---
+`value` | `CustomizableRadioValue` | An array that defines a set of radio buttons
+
+### CustomizableRadioValue object
```
Comment (keharper) [inline guides/v2.3/graphql/reference/customizable-option-interface.md]: Lines 159 - 170 are duplicates of 140 - 153.

File: guides/v2.3/graphql/reference/customizable-option-interface.md [hunk 2]
```diff
@@ -140,6 +141,46 @@ Attribute | Type | Description
...
+
+`CustomizableCheckboxOption` contains information about a set of checkbox values that are defined as part of a customizable option.
+
+Attribute | Type | Description
+--- | --- | ---
+`value` | `CustomizableCheckboxValue` | An array that defines a set of checkbox values
+
+### CustomizableCheckboxValue object
+
+`CustomizableCheckboxValue`  defines the price and sku of a product whose page contains a customized set of checkbox values.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/customizable-option-interface.md]:
```suggestion
`CustomizableCheckboxValue`  defines the attributes of a product whose page contains a customized set of checkbox values.
```

**Comment by ParthBrahmbhatt** [comment]
Hello @keharper 
I have updated the changes suggested by you.

**Comment by keharper** [comment]
@ParthBrahmbhatt Thank you for addressing my inline comments, but you missed this one:

> The CustomizableMultipleOption and CustomizableMultipleValue types were also added for 2.3.2. Please add these to your PR.

**Comment by ParthBrahmbhatt** [comment]
Hi @keharper 
I have also added the missed suggetion related to CustomizableMultiple.

**Comment by ParthBrahmbhatt** [comment]
Hello @keharper 
I have added suggested text into the configuration option list

**Comment by keharper** [comment]
Thanks @ParthBrahmbhatt

**Pull request merged**